Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -130,14 +153,17 @@ private void OnDeviceManagerDeviceOptionsUpdated(object sender, GenericEventArgs
             }
         }
 
-        private bool _disposed = false;
-
+        /// <inheritdoc />
         public void Dispose()
         {
             Dispose(true);
             GC.SuppressFinalize(this);
         }
 
+        /// <summary>
+        /// Releases unmanaged and - optionally - managed resources.
```

Comment:
```suggestion
        /// Releases unmanaged and optionally managed resources.
```

Reply:
This is copy pasted from elsewhere and exists in multiple places, lets keep it consistent

Reply:
I've been updating this in other places too so it will eventually be consistent again.